packaging/build_conda.sh

```diff
@@ -12,4 +12,4 @@ setup_conda_cudatoolkit_constraint
 setup_visual_studio_constraint
 setup_junit_results_folder
 # nvidia channel included for cudatoolkit >= 11
-conda build -c defaults -c nvidia $CONDA_CHANNEL_FLAGS --no-anaconda-upload --python "$PYTHON_VERSION" packaging/torchvision
+conda build -c defaults -c conda-forge $CONDA_CHANNEL_FLAGS --no-anaconda-upload --python "$PYTHON_VERSION" packaging/torchvision
```

Comment:
We should be wary about adding `conda-forge`, iirc this causes a ton more problems with dependencies

Reply:
Context, https://github.com/pytorch/vision/pull/4082